### DISCUSSION TITLE
Seconder Metabolite Finding

### DISCUSSION THREAD

**Discussion by airbender97** [jotech/gapseq#138] [Ideas]
Hello,

Can we find pathways and other related things for seconder metabolites? Is so, can you please help me?

**Comment by jotech** [comment]
Hi @airbender97 
are you looking for secondary metabolites?
By default, gapseq looks for [biosynthesis](https://metacyc.org/META/NEW-IMAGE?type=ECOCYC-CLASS&object=SECONDARY-METABOLITE-BIOSYNTHESIS) and [degradation](https://metacyc.org/META/NEW-IMAGE?type=ECOCYC-CLASS&object=SECONDARY-METABOLITE-DEGRADATION) pathways of secondary metabolites from metacyc.

**Comment by airbender97** [comment]
I am looking for pathways, but I don't know how can I run a code for them.

**Comment by jotech** [comment]
You can check the output files if you already have a gapseq annotation. 

```R
library(data.table)
meta <- fread("gapseq/dat/meta_pwy.tbl")
sm.pwy <- meta$id[grep("SECONDARY-METABOLITE-BIOSYNTHESIS", meta$hierarchy)]
eco.gs <- fread("gapseq/toy/myb71-all-Pathways.tbl")
eco.gs[eco.gs$ID %in% sm.pwy]
```

Alternatively, you can also have a look at sm pathways selectively.
```
./gapseq find -p "SECONDARY-METABOLITE-BIOSYNTHESIS" toy/ecoli.fna.gz
```
